.eslintrc.yaml

```diff
@@ -17,7 +17,23 @@ plugins:
   - react
   - react-hooks
   - testing-library
+rules:
+  indent: [error, 2]
+  quotes:
+    - error
+    - single
+    - avoidEscape: true
+      allowTemplateLiterals: true
```

Comment:
__*Preference*__
9/10

Instead of putting formatting rules in eslint use prettier to check formatting. Use [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier/). That way we don't end up in a situation where they conflict with one another.

Reply:
Sounds great to me. This was like this when I moved it (I just reordered it so the overrides were left at the bottom).

I'll do it in a separate PR to keep things as tidy as possible!